.vscode/settings.json

```diff
@@ -3,10 +3,15 @@
         120
     ],
     "editor.formatOnSave": true,
```

Comment:
should we delete this one as well since its wrapped within [python] > editor.formatOnSave?

Reply:
There are other non-python files that this is useful for like JSON.